spec/models/remote_theme_spec.rb

```diff
@@ -468,6 +468,136 @@ def about_json(
       end
     end
 
+    describe "color schemes" do
+      it "updates the base schemes for schemes that have diverged colors" do
+        add_to_git_repo(
+          initial_repo,
+          "about.json" =>
+            JSON
+              .parse(about_json)
+              .merge(
+                color_schemes: {
+                  scheme1: {
+                    love: "FFFFFF",
+                    tertiary_low: "000000",
+                  },
+                  scheme2: {
+                    love: "AACCDD",
+                    tertiary_low: "99AAFF",
+                  },
+                },
+              )
+              .to_json,
+        )
+
+        theme = RemoteTheme.import_theme(initial_repo_url)
+        scheme1 = theme.color_schemes.find_by(name: "scheme1")
+        scheme2 = theme.color_schemes.find_by(name: "scheme1")
+        expect(scheme1.base_scheme_id).to eq(nil)
+        expect(scheme2.base_scheme_id).to eq(nil)
+
+        expect do
+          ColorSchemeRevisor.revise(
+            scheme1,
+            { colors: [{ name: "love", hex: "111111" }] },
+            diverge_from_remote: true,
+          )
+        end.to change { ColorScheme.unscoped.count }.by(1)
```

Comment:
```suggestion
        end.to change { scheme1.base_scheme.exists? }.from(false).to(true)
```

I think the above makes it clearer about what we are expecting.